azimuth/modules/base_classes/caching.py

```diff
@@ -101,6 +107,10 @@ def _store_data_in_cache(self, result: List[ModuleResponse], indices: List[int])
                 ds[()] = arr
             handle.flush()
 
+        # Save all that affects caching so it can be used for identification and debugging.
+        with open(pjoin(self._cache_metadata), "w+") as f:
+            json.dump(self.get_meta_data().no_alias_dict(), f, indent=4)
```

Comment:
In our repo, we format JSON files with 2 indents, so I would suggest
```suggestion
            json.dump(self.get_meta_data().no_alias_dict(), f, indent=2)
```